django_rq/workers.py

```diff
@@ -23,3 +23,20 @@ def get_worker(*queue_names):
     return Worker(queues,
                   connection=queues[0].connection,
                   exception_handlers=get_exception_handlers() or None)
+
+
+def collect_workers_per_configuration(queue_list):
```

Comment:
Can we rename this to `collect_workers_by_connection(queues)`? I think it's more elegant if this returns a dictionary of workers keyed by its connection config like this:
```python
{
    CONFIG_1: [worker1],
    CONFIG_2: [worker2],
}
```
We can then use this in `views.py` as such:
```python
workers_dict = collect_workers_by_connection(QUEUES_LIST)

for index, config in enumerate(QUEUES_LIST):
    all_workers = workers_dict[config['connection_config']]
```